### ISSUE TITLE
Duplicate FormKey created by using DialogResponse IModContext.GetOrAddAsOverride

### ISSUE THREAD

**Issue by mrudat** [Mutagen-Modding/Mutagen#173]
Duplicate FormKey created by using `IModContext.GetOrAddAsOverride` for a DialogResponse at https://github.com/mrudat/RaceCompatibilityDialogue/blob/main/RaceCompatibilityDialogue/Program.cs#L76.

I boldly assume I'm doing this the Right Way.

```
System.ArgumentException: Two records with the same FormKey were encountered: 177AE7:Alternate Start - Live Another Life.esp
   at Mutagen.Bethesda.Internals.ModHeaderWriteLogic.<AddFormIDUniqueness>b__17_0(IMajorRecordCommonGetter maj)
   at Mutagen.Bethesda.Internals.ModHeaderWriteLogic.RunProcessors(IModGetter mod)
   at Mutagen.Bethesda.Internals.ModHeaderWriteLogic.WriteHeader(BinaryWriteParameters param, MutagenWriter writer, IModGetter mod, IModHeaderCommon modHeader, ModKey modKey)
   at Mutagen.Bethesda.Skyrim.Internals.SkyrimModCommon.WriteParallel(ISkyrimModGetter item, Stream stream, BinaryWriteParameters param, ModKey modKey)
   at Mutagen.Bethesda.Skyrim.SkyrimModMixIn.WriteToBinaryParallel(ISkyrimModGetter item, String path, BinaryWriteParameters param)
   at Mutagen.Bethesda.Skyrim.SkyrimMod.Mutagen.Bethesda.IModGetter.WriteToBinaryParallel(String path, BinaryWriteParameters param)
   at Mutagen.Bethesda.Synthesis.SynthesisPipeline.Run(RunSynthesisMutagenPatcher args, ModKey exportKey, RunPreferences preferences)
   at Mutagen.Bethesda.Synthesis.SynthesisPipeline.Run(String[] args, RunPreferences preferences)
```

**Comment by Noggog** [comment]
Yeah, looking like a potential bug at first glance.  I'll dig in after work and find out more

**Comment by Noggog** [comment]
Should be fixed /w 0.25.1
Let me know how it goes.  Feel free to reopen if it doesn't look right
